health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -89,18 +198,120 @@ const FormulaConfiguration = ({onSelect,  category,  customProps})=>{
+                      title=""
+                      variation="secondary"
+                      isDisabled={isAddNewDisabled}
+                      
+                    />
+                  {showPopUP && <PopUp
+                                    className={"popUpClass"}
+                                    type={"default"}
+                                    heading={t("CONFIRM_TO_DELETE")}
+                                    equalWidthButtons={true}
+                                      children={[
+                                        <div>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)